<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWizard</class>
 <widget class="QWizard" name="MainWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MessageMerge</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::DisabledBackButtonOnLastPage|QWizard::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="startPage">
   <property name="title">
    <string>Template &amp; Contacts</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Write Template:</string>
      </property>
      <property name="buddy">
       <cstring>templateEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QPlainTextEdit" name="templateEdit">
      <property name="plainText">
       <string>Hello [[firstname]] !</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="loadTemplateBtn">
      <property name="text">
       <string>&amp;Load...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QPushButton" name="saveAsTemplateBtn">
      <property name="text">
       <string>&amp;Save As...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="editTemplatesBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Templates...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Choose Contacts:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QListWidget" name="contactList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="previewPage">
   <property name="title">
    <string>Preview Messages</string>
   </property>
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Show for:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="previewContactCombo"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="messagePreviewEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="generatePage">
   <property name="title">
    <string>Generate</string>
   </property>
   <attribute name="pageId">
    <string notr="true">3</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QRadioButton" name="saveFilesRadio">
      <property name="text">
       <string>Save as files to folder:</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="folderEdit">
      <property name="text">
       <string>C:\MessageMerge</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="sendTextRadio">
      <property name="text">
       <string>Create and Send Text Messages</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="draftTextRadio">
      <property name="text">
       <string>Create Text Messages and Save as Draft</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>234</width>
        <height>126</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="processingPage">
   <property name="title">
    <string>Processing</string>
   </property>
   <property name="subTitle">
    <string>Please wait...</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="processingLabel1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>In your case you would have to set up the connection manually, but you shouldn't use the &quot;on_&quot; prefix for the slot's name because connectSlotsByName is also called at the end of setupUi which would again throw the warning that there is no matching signal for the slot.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>78</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="processingLabel2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="finishPage">
   <property name="title">
    <string>Finished.</string>
   </property>
   <property name="subTitle">
    <string>Processing has completed.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="finishLabel">
      <property name="text">
       <string>MessageMerge has finished processing.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>previewContactCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWizard</receiver>
   <slot>refreshMessagePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshMessagePreview()</slot>
 </slots>
</ui>
